lutris/api.py

```diff
@@ -501,7 +501,6 @@ def format_installer_url(installer_info):
     if launch_config_name:
         parts.append(launch_config_name)
 
-    parts = [urllib.parse.quote(str(part)) for part in parts]
```

Comment:
Here we might need some code to check for specific runners. I'm unsure if this format might be needed by shortcuts for other runners then `wine` and `steam` to properly work. *.desktop files break when this code is used.

Reply:
This should not be runner specific; the thing that needs to be quoted is the launcher option name. We don't want that to be understood as multiple path segments in the URL.

I do not understand the bug here; we should be unquoting everything before use.

Reply:
> This should not be runner specific; the thing that needs to be quoted is the launcher option name. We don't want that to be understood as multiple path segments in the URL.
> 
> I do not understand the bug here; we should be unquoting everything before use.

You're right. 

The bug is that *.desktop files currently won't work when we create a shortcut of an launch config option using the `LaunchConfigSelectDialog`.